### PR TITLE
clarify spaces behavior

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
This clarifies statements regarding spaces

### CODE REVIEW

File: correlation_context/HTTP_HEADER_FORMAT.md [hunk 1]
```diff
@@ -23,15 +23,17 @@ Multiple correlation context headers are allowed. Values can be combined in a si
...
 
-Url encoded string. Spaces should be trimmed from beginning and the end of the name. Names are case sensitive.
+Url encoded string. Spaces are allows before and after the name. Header with the trimmed name and with spaces before and after name MUST be considered identical.
 
 ## Value format
 
-All spaces should be trimmed from the beginning and the end of the value. Value ends with the special character `;`, separator `,` or end of string. Value represents a url encoded string and case sensitive. 
+Value starts after equal sign and ends with the special character `;`, separator `,` or end of string. Value represents a url encoded string and case sensitive. Spaces are allowed in the beginning and the end of the value. Value with spaces before and after MUST be considered identical to the trimmed value. 
 
 ## Properties
```
Comment (bogdandrutu) [inline correlation_context/HTTP_HEADER_FORMAT.md:32]: Do we have a requirement to preserve or not the order? I think it is fine to say that the order of the properties may be changed. Same comment for trace-state probably.

Comment (SergeyKanzhelev) [reply correlation_context/HTTP_HEADER_FORMAT.md:32]: Let's keep this separate. This PR is about spaces behavior

Comment (SergeyKanzhelev) [reply correlation_context/HTTP_HEADER_FORMAT.md:32]: https://github.com/w3c/distributed-tracing/issues/71

Comment (SergeyKanzhelev) [reply correlation_context/HTTP_HEADER_FORMAT.md:32]: Can you please approve this one?

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**